### PR TITLE
Fix Change products by components

### PR DESCRIPTION
**Pull request by aljesusg**
```json
{
  "status": {
    "Kiali console version": "0.2.10-SNAPSHOT.1213",
    "Kiali core commit hash": "91d178374fa13f6ff932d9a04bd94bf19f62eb02",
    "Kiali core version": "0.3.2.Alpha-SNAPSHOT",
    "Kiali state": "running"
  },
  "externalServices": [
    {
      "name": "Istio",
      "version": "0.7.1"
    },
    {
      "name": "Prometheus",
      "version": "2.0.0"
    },
    {
      "name": "Kubernetes",
      "version": "v1.7.6+a08f5eeb62"
    }
  ],
  "warningMessages": []
}

```

### CODE REVIEW

**Comment by jotak** [comment]
What's the reason exactly for that? Is it because "products" sounds inaccurate? Maybe something like "external service" would be even more accurate than "component", no? (although more verbose)

**Comment by aljesusg** [comment]
We discussed about this in https://github.com/kiali/kiali-ui/pull/404 , and it's better call them component than products @jotak

**Comment by jotak** [comment]
@aljesusg ok, I've read it. But I think "component" refers in general to something internal. I wouldn't call prometheus or istio as "components" of Kiali.
Wording is a hard task :)

**Comment by aljesusg** [comment]
Yes but they are not products of Kiali, that is the point. Maybe we can discuss this in the meeting tomorrow or IRC with the team

**Comment by jotak** [comment]
@aljesusg yeah... it's too late to have clear ideas now :)

**Comment by xeviknal** [comment]
+1 to dependencies although external services looks good too!

**Comment by aljesusg** [comment]
@lucasponce @abonas What do you think about use dependencies instead components?

**Comment by lucasponce** [comment]
"Dependencies" can be also understood as something internal too.

Looking in the first screenshot, changing "components" by "services" (or 'external services' if we want something very orthodox) works as well.

"Prometheus service version x.y.z" makes perfectly sense to me as well.

**Comment by aljesusg** [comment]
So externalService and Json return key service?

File: README.adoc [hunk 1]
```diff
@@ -297,15 +297,15 @@ service_filter_label_name: VALUE
 |The URL used to access and query the Prometheus Server.
 [source,yaml]
 ----
-products:
+components:
```
Comment (lucasponce) [inline README.adoc]: If we finish up with ExternalServices I guess it should be updated in the doc as well.

File: config/config.go [hunk 2]
```diff
@@ -109,7 +109,7 @@ type Config struct {
 	InCluster              bool              `yaml:"in_cluster,omitempty"`
 	ServiceFilterLabelName string            `yaml:"service_filter_label_name,omitempty"`
 	VersionFilterLabelName string            `yaml:"version_filter_label_name,omitempty"`
-	Products               Products          `yaml:"products,omitempty"`
+	ExternalServices       ExternalServices  `yaml:"components,omitempty"`
```
Comment (lucasponce) [inline config/config.go]: better to unify the json/yaml too.

**Comment by lucasponce** [comment]
ExternalServices look good to me as it close what is represents.
I put a couple of comments to unify the wording.

**Comment by aljesusg** [comment]
@lucasponce @jotak @xeviknal Ready for merge, Thanks. We should merge https://github.com/kiali/kiali-ui/pull/424 after to keep the aboutpage working in the UI

**Pull request merged**